apps/storybook.namekit.io/stories/Namekit/Tooltip.stories.tsx

```diff
@@ -45,6 +71,28 @@ export const LeftPlacement: Story = {
     trigger: <>Tooltip Trigger</>,
     children: <>Tooltip content</>,
   },
+  render: (args) => {
+    return (
+      <div className="nk-w-full nk-flex nk-justify-center">
+        <Tooltip {...args} />
+      </div>
+    );
+  },
+};
+
+export const TopPlacement: Story = {
```

Comment:
You should move `TopPlacement` to come before `BottomPlacement`. People normally think in order from top, bottom, left, right.